### ISSUE TITLE
Error with getAssets when execute ./gradlew setupDecompWorkspace

### ISSUE THREAD

**Issue by wangst0181** [MineDojo/MineDojo#53]
Found AccessTransformer: malmomod_at.cfg
Error downloading asset (try 1) : icons/icon_32x32.png
Error downloading asset (try 1) : minecraft/icons/icon_32x32.png
Error downloading asset (try 1) : minecraft/lang/ar_sa.lang
Error downloading asset (try 1) : minecraft/lang/af_za.lang
java.io.IOException: Server returned HTTP response code: 400 for URL: http://resources.download.minecraft.net/92/92750c5f93c312ba9ab413d546f32190c56d6f1f
Error downloading asset (try 1) : icons/icon_16x16.png
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1914)
Error downloading asset (try 1) : minecraft/icons/minecraft.icns
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1512)
Error downloading asset (try 1) : icons/minecraft.icns
        at java.net.URL.openStream(URL.java:1092)
        at net.minecraftforge.gradle.tasks.DownloadAssetsTask$GetAssetTask.call(DownloadAssetsTask.java:189)
        at net.minecraftforge.gradle.tasks.DownloadAssetsTask$GetAssetTask.call(DownloadAssetsTask.java:150)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:750)
java.io.IOException: Server returned HTTP response code: 400 for URL: http://resources.download.minecraft.net/99/991b421dfd401f115241601b2b373140a8d78572
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1914)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1512)
        at java.net.URL.openStream(URL.java:1092)
        at net.minecraftforge.gradle.tasks.DownloadAssetsTask$GetAssetTask.call(DownloadAssetsTask.java:189)
        at net.minecraftforge.gradle.tasks.DownloadAssetsTask$GetAssetTask.call(DownloadAssetsTask.java:150)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:750)

......

**the url provided above is invalid, information below offered when visit the url from browser:** 
This XML file does not appear to have any style information associated with it. The document tree is shown below.
```
<Error>
<Code>AccountRequiresHttps</Code>
<Message>The account being accessed does not support http. RequestId:325f0e43-e01e-0025-7ad8-455960000000 Time:2023-02-21T09:37:52.2562539Z</Message>
<AccountName>resourcesdownloadminecra</AccountName>
</Error>
```

**Comment by wangst0181** [comment]
> The download links should be start with "https://" instead of "http://" . However, I have not find the way to reapir it.

OMG... it's true, thx!!! At least it's a right way. However, I've got no idea how to fix it too.

**Comment by yunfanjiang** [comment]
Hi @wangst0181, would you mind sharing more information about your runtime environment? E.g., the Java version, your OS info, etc. We can better help with this additional information.

**Comment by yunfanjiang** [comment]
We have addressed this issue as in 8d5c6dd. Feel free to let us know if there are other problems.

**Comment by wangst0181** [comment]
> We have addressed this issue as in [8d5c6dd](https://github.com/MineDojo/MineDojo/commit/8d5c6ddcad147fdf0f58cf91ccf3611d480dff5c). Feel free to let us know if there are other problems.

Thanks!!!!!!!
